content/sensu-core/0.29/guides/securing-redis-guide.md

```diff
@@ -0,0 +1,70 @@
+---
+title: "Securing Redis"
+description: "Strategies and best practices for security Redis"
+version: "0.29"
+weight: 9
+next: ../securing-rabbitmq-guide
+previous: ../securing-sensu
+menu:
+  sensu-core-0.29:
+    parent: guides
+---
+
+Redis is a key-value database, which describes itself as “an open source, BSD licensed, advanced key-value cache and store”. Sensu uses Redis for storing persistent data. Two Sensu services, the server and API, require access to the same instance of Redis to function.
+
+This guide will discuss best practices to use with Redis for use with Sensu.
+
+### Objectives
+
+This guide will discuss the following:
+
+* [Redis General Security Model](#redis-general-security-model)
+* [Securing Redis with a Local Install](#securing-redis-with-a-local-install)
+* [Securing Redis via Localhost Security](#securing-redis-localhost-security)
+
+## Redis General Security Model{#redis-general-security-model}
+
+Redis was designed to be accessed by trusted clients inside a closed network environment. As such it is recommended that Redis instances not be directly exposed to the internet or have access in general to untrusted clients that can directly connect to the Redis TCP port or UNIX socket.
+
+Best practices from [Redis Security Documentation][1] suggest blocking port level access to all hosts except trusted hosts, in our case your Sensu-Server, Sensu-API and/or Sensu-Enterprise-Server.
+
+_NOTE: As of [Sensu 1.3.0][2], TLS is now supported, allowing you to encrypt your traffic between Sensu and Redis when being used as a Transport or Datastore._
+
+
+## Securing Redis with a Local Installation of Sensu{#securing-redis-with-a-local-install}
+
+For instances where you will be running Redis on the same host that you will be running Sensu, you can configure Redis to listen to the localhost only on the host loopback IP address.
+
+To accomplish this you would need to edit `/etc/redis/redis.conf` with following line:
```

Comment:
- Switch `would` to `will` to stay consistent with the tense you use in the rest of the guide.
- Missing `the`: `with the following line`